cloudera-integration/csd/scripts/control.sh

```diff
@@ -40,6 +40,24 @@ export LOGBACK_CONFIG=${LOGBACK_CONFIG:-$DEFAULT_LOGBACK_CONFIG}
 export KEYTAB_FILE=${KEYTAB_FILE:-"${CONF_DIR}/pulse.keytab"}
 export AKKA_CONF=${AKKA_CONF:-"application.conf"}
 
+if [ -f "Path to file containing COLLECTION_ROLLER_CONFIG" ]; then #Checking if file exists
+    if [ ! -s "Path to file containing COLLECTION_ROLLER_CONFIG" ]; then #Checking if file is not empty
+        export DEFAULT_COLLECTION_ROLLER_CONFIG=$(readlink -f cloudera-integration/csd/default-configs/default-collection-roller.yml)
```

Comment:
These are all used locally no no reason to use `export`

Reply:
This path `cloudera-integration/csd/default-configs/` isn't going to exist when running under a CSD. After the CSD is unpacked it'd probably be `default-configs/default-collection-roller.yml` or you could include it inline.

Reply:
Based on the commit, I think it'd be in `aux/default-alert-engine.yml`

Reply:
Yup you're right, I'll update the path to aux/default-alert-engine.yml